File: VisaChecker.UI/src/firebase.js
import { getDatabase } from 'firebase/database'
// Import the functions you need from the SDKs you need
import { initializeApp } from "firebase/app";
// TODO: Add SDKs for Firebase products that you want to use
// https://firebase.google.com/docs/web/setup#available-libraries

// Your web app's Firebase configuration
const firebaseConfig = {
  apiKey: "AIzaSyCBdpiFmk7oLZsLal-zBDYkm40OO2-hw9w",
  authDomain: "mvcr-database.firebaseapp.com",
  databaseURL: "https://mvcr-database-default-rtdb.europe-west1.firebasedatabase.app",
  projectId: "mvcr-database",
  storageBucket: "mvcr-database.appspot.com",
  messagingSenderId: "389674513041",
  appId: "1:389674513041:web:7eb4d6875a7b60bbd44530"
};

// Initialize Firebase
const app = initializeApp(firebaseConfig);

export const database = getDatabase(app);

File: VisaChecker.Server/src/main/java/Firebase.java
package main.java;

import com.google.auth.oauth2.GoogleCredentials;
import com.google.firebase.FirebaseApp;
import com.google.firebase.FirebaseOptions;

import java.io.FileInputStream;
import java.io.IOException;

//Initializing connection to Firebase
public class Firebase {
    
    final static String databaseUrl = "https://mvcr-database-default-rtdb.europe-west1.firebasedatabase.app";

    public static void initialize() throws IOException {
        FileInputStream serviceAccount =
                new FileInputStream("./ServiceAccountKeyVisa.json");

        FirebaseOptions options = new FirebaseOptions.Builder()
                .setCredentials(GoogleCredentials.fromStream(serviceAccount))
                .setDatabaseUrl(databaseUrl)
                .build();

        FirebaseApp.initializeApp(options);
    }
}
